static/redfish/v1/$metadata/index.xml

```diff
@@ -3743,6 +3743,9 @@
         <edmx:Include Namespace="OemLogEntry"/>
         <edmx:Include Namespace="OemLogEntry.v1_0_0"/>
     </edmx:Reference>
+    <edmx:Reference Uri="/redfish/v1/schema/OemLogEntryAttachment_v1.xml">
+        <edmx:Include Namespace="LogEntryAttachment"/>
```

Comment:
I would think this would be 
         <edmx:Include Namespace="OemLogEntryAttachment"/>
         <edmx:Include Namespace="OemLogEntryAttachment.v1_0_0.xml"/>

Reply:
Done.